### ISSUE TITLE
There is an error  in the 'nested resampling' section of the documentation.

### ISSUE THREAD

**Issue by skanskan** [mlr-org/mlr3book#721]
### Description

Hello.

I think there is an error in the mlr3 book, at the 'nested resampling' section.
Where it explains Figure 4.5.

<pre>
1. Outer resampling start...
2. Inner resampling...
<b>3. HPO – Tune the hyperparameters on the outer training set (large, light blue blocks) using the inner data splits.</b>
4. Training...
</pre>

Shouldn't the third point say "on the inner training set"?

https://mlr3book.mlr-org.com/chapters/chapter4/hyperparameter_optimization.html#sec-nested-resampling

### Reproducible example

https://mlr3book.mlr-org.com/chapters/chapter4/hyperparameter_optimization.html#sec-nested-resampling

**Comment by larskotthoff** [comment]
So the outer training set is what's partitioned in the inner resampling, and that's exactly what you want to tune on. This is then evaluated on the outer test set. The inner training set is used to train the models for a particular hyperparameter setting. That is, no model is actually trained on the entire outer training set, as it is split up further.

Does that make it clearer?

**Comment by RaphaelS1** [comment]
Closing for now - let us know if your question isn't answered and we can re-open
